src/project.test.js

```diff
@@ -2,32 +2,81 @@ const { project } = require('../');
 
 test('valid projection', () => {
   expect(
-    project({
-      forestEcoregion: '1',
-      additional: 'unknown',
-      heightLevel: 'SA',
-      forestType: '60*',
-      slope: '<70',
-      tannenareal: 'unknown',
-      relief: 'unknown',
-    }).target,
-  ).toBe('50*');
+    project(
+      {
+        forestEcoregion: '1',
+        additional: 'unknown',
+        heightLevel: 'OM',
+        forestType: '1h',
+        slope: 'unknown',
+        tannenareal: 'unknown',
+        relief: 'unknown',
+      },
+      'UM',
+    ).forestType,
+  ).toBe('1');
+});
+
+test('valid multi heightLevel projection', () => {
+  expect(
+    project(
+      {
+        forestEcoregion: '3',
+        additional: 'unknown',
+        heightLevel: 'OSA',
+        forestType: '59L',
+        slope: 'unknown',
+        tannenareal: 'unknown',
+        relief: 'unknown',
+      },
+      'OM',
+    ).forestType,
+  ).toBe('1h');
+});
+
+test('valid data type for projection', () => {
```

Comment:
This case is already covered by the first test. Instead we should replace this one with a test which catches 'Found no projection for selected target height.' (use `toThrowError`).